go.mod

```diff
@@ -181,3 +181,5 @@ replace k8s.io/code-generator => k8s.io/code-generator v0.31.13 //allow-merging
 replace k8s.io/component-base => k8s.io/component-base v0.31.13 //allow-merging
 
 replace github.com/cert-manager/cmctl/v2 => github.com/cert-manager/cmctl/v2 v2.1.2-0.20241127223932-88edb96860cf //allow-merging
+
+replace github.com/openstack-k8s-operators/cinder-operator/api => github.com/fmount/cinder-operator/api v0.1.1-0.20251129140305-509169b1ebe0
```

Comment:
will remove this after we land https://github.com/openstack-k8s-operators/cinder-operator/pull/575